### DISCUSSION TITLE
Ringless voicemail drops support in Azure communication Call Automation SDK

### DISCUSSION THREAD

**Discussion by raviraj1976** [Azure/communication-ui-library#3472] [General]
Please let us know ACS roadmap to support Ringless voicemail drops i.e. Call directly to a voicemail system and send ringless voicemail drop messages directly to your contacts similar to https://www.slybroadcast.com/#hiw

Thanks

Raj

**Comment by RyanGentry198** [comment]
If you're looking for better ringless voicemail solutions, check out https://www.dropcowboy.com/ 
 — it's a Tier 1 product."
